node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/container/CGroupV2.java

```diff
@@ -27,12 +28,43 @@ public class CGroupV2 {
 
     private static final Logger logger = Logger.getLogger(CGroupV2.class.getName());
     private static final String MAX = "max";
-    public static final String VESPA_CGEXEC_PATH = "/opt/vespa/bin/vespa-cgexec";
 
-    private final FileSystem fileSystem;
+    private final Path rootCgroupPath;
 
     public CGroupV2(FileSystem fileSystem) {
-        this.fileSystem = fileSystem;
+        this.rootCgroupPath = fileSystem.getPath("/sys/fs/cgroup");
+    }
+
+    /**
+     * Wraps {@code command} to ensure it is executed in the given cgroup.
+     *
+     * <p>WARNING: This method must be called only after vespa-cgexec has been installed.</p>
+     *
+     * @param cgroup  The cgroup to execute the command in, e.g. system.slice/wireguard.scope.
+     * @param command The command to execute in the cgroup.
+     * @see #cgroupOf(ContainerId)
+     */
+    public String[] wrapForExecutionIn(String cgroup, String... command) {
+        UnixPath path = new UnixPath(rootCgroupPath.resolve(cgroup).normalize());
+        if (!path.toString().startsWith(rootCgroupPath + "/"))
```

Comment:
Shouldn't `vespa-cgexec` check this?

Reply:
OK.  It will add another dependency of the script: `realpath`, which is installed on a sample host in AWS.  It would be very strange if that is not installed (to my knowledge only MacOS doesn't have it).

Reply:
And, I added a `mkdir` if cgroup does not already exist, and also allows passing a /sys/fs/cgroup path.